src/accountScreen/accountScreen.css

```diff
@@ -0,0 +1,31 @@
+.rectangle-4 {
+    height: 100vh;
+    width: 411px;
+    background-color: #FFFFFF;
+    box-shadow: 0 2px 12px 3px rgba(0, 0, 0, 0.14);
+}
+
+.rectangle-7 {
+    height: 43px;
+    width: 345px;
+    border: 1px solid #979797;
+    border-radius: 21.5px;
+    margin-top: 55px;
+}
+.search-chats {
+    height: 23px;
+    width: 101px;
+    color: #706F6F;
+    font-family: "Avenir Next";
+    font-size: 17px;
+    line-height: 23px;
+}
+
+.searchContainer{

```

Comment:
I see you're switching between conventions here. 
`search-container`?

Reply:
corrected conventions. Noted